docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md

```diff
@@ -32,22 +32,26 @@ showAsideToc: true
 
 You can use YugabyteDB Anywhere to back up your YugabyteDB universe YCQL data.
 
-To schedule backups for a later time or as a recurring task, see [Schedule universe YCQL data backups](../../schedule-data-backups/ycql).
+To view, [restore](../../restore-universe-data/ycql/), or delete existing backups for your universe, navigate to that universe and select **Backups**, as per the following illustration:
 
-To immediately back up your YugabyteDB universe YCQL data, perform the following:
+![Create Backup](/images/yp/create-backup-new-1.png)
 
-1. Navigate to your universe and select **Backups**.
+To view details information about an existing backup, click on it to open **Backup Details**.
 
-1. Click **Create Backup** to open the **Create Backup** dialog shown in the following illustration:
+The **Backups** page allows you to create new backups that start immediately, as follows: 
 
-    <br/><br/>
+- Select the time period for the backup and click **Backup now** to open the dialog shown in the following illustration:<br><br>
```

Comment:
What is the time period that you are referring to here?

Reply:
I misunderstood the time period selection box. Fixed the text now.